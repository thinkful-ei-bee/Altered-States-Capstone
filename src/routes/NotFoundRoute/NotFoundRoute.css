.not-found {
  margin-top: 100px;
}

.not-found img {
  width: 100px;
}