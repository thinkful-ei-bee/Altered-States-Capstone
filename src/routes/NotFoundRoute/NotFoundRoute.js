import React, { Component } from 'react'
import './NotFoundRoute.css';
import verySad from '../../images/verySad.png'

class NotFoundRoute extends Component {
  render() {
    return (
      <section className='not-found'>
        <img src={verySad} alt='verySad' />
        <h2>404 - Page not found</h2>
        <p>Try going back to the previous page.</p>
      </section>
    );
  }
}

export default NotFoundRoute
