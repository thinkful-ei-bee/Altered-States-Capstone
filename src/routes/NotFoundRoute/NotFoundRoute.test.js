import React from 'react';
import NotFoundRoute from './NotFoundRoute';
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

Enzyme.configure({ adapter: new Adapter() })

describe('DashboardRoute', () => {

  it('renders the complete page', () => {
    const wrapper = shallow(<NotFoundRoute />);
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})