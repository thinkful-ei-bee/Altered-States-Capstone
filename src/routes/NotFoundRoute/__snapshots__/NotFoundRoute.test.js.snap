// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardRoute renders the complete page 1`] = `
<section>
  <h2>
    404 - Page not found
  </h2>
  <p>
    Try going back to the previous page.
  </p>
</section>
`;
